docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: "Quickstart: Run your dbt project with Dagster in 3 steps"
+description: Get started swiftly with this simple dbt + Dagster example.
+---
+
+# Quickstart: Run your dbt project with Dagster in 3 steps
+
+---
+
+## Step 1: Setup your environment
+
+<Note>
+  <strong>Note</strong>: We strongly recommend installing Dagster inside a
+  Python virtualenv. Visit the{" "}
+  <a href="/getting-started/install">Dagster installation docs</a> for more
+  info.
+</Note>
+
+Install dbt, Dagster, and the Dagster webserver/UI\*\*. Run the following to install everything using pip:
+
+```shell
+pip install dagster-dbt dagster-webserver
+```
+
+The `dagster-dbt` library installs both `dbt-core` and `dagster` as dependencies. Refer to the [dbt](https://docs.getdbt.com/dbt-cli/install/overview) and [Dagster](/getting-started/install) installation docs for more info.
+
+---
+
+## Step 2: Create your code location file
+
+Running your dbt project with Dagster can be easily done after creating a single file. For this example, let's consider a basic use case - say you want to represent your dbt models as Dagster assets and run them daily at midnight.
+
+With your text editor of choice, create a Python file and add the following code. Make sure to update the `RELATIVE_PATH_TO_YOUR_DBT_PROJECT` constant so that it points to your dbt project. Note that since this file contains [all Dagster definitions required for your code location](/concepts/code-locations), it is recommended to name it `definitions.py`.
+
+<Note>
+  <strong>Note</strong>: This code uses the <code>DbtProject</code> class,
+  which is an experimental feature. Visit the{" "}
+  <a href="/\_apidocs/libraries/dagster-dbt">dagster-dbt library API reference</a> for more
+  info.
+</Note>
+
+```python file=/integrations/dbt/quickstart.py startafter=start_example endbefore=end_example
+from pathlib import Path
+
+from dagster import AssetExecutionContext, Definitions
+from dagster_dbt import (
+    DbtCliResource,
+    DbtProject,
+    build_schedule_from_dbt_selection,
+    dbt_assets,
+)
+
+RELATIVE_PATH_TO_YOUR_DBT_PROJECT = "relative/path/to/your/dbt-project"
```

Comment:
Hmm, I think our users might benefit from some additional hand-holding here.

How do you feel about either:
- instructing them to make this file in/adjacent to the dbt project dir?
- or giving a more precise example of what the path could be?